backend/src/bundles/avatar/avatar.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpError } from '~/common/http/http.js';
+import { type AzureAIService } from '~/common/services/azure/azure-ai.service.js';
+
+import { type GetAvatarVoicesResponseDto } from './types/types.js';
+
+class AvatarService {
+    private azureAIService: AzureAIService;
+
+    public constructor(azureAIService: AzureAIService) {
+        this.azureAIService = azureAIService;
+    }
+
+    public async getVoices(): Promise<GetAvatarVoicesResponseDto[]> {
+        try {
+            return await this.azureAIService.getAvatarVoices();
+        } catch (error) {
+            throw new HttpError({
+                message: 'Failed to fetch voices',
+                status: 500,
```

Comment:
1. Use HttpCode enum values
2. Internal Server Errors are handled automatically in [BaseServerApp](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/blob/next/backend/src/common/server-application/base-server-app.ts). It is better to add custom messages enum. Use unified approach all over the project

Reply:
deleted internal server error and changed using HttpCode. Also with custom message enum